Tests/Functional/ServiceInstantiationTest.php

```diff
@@ -21,6 +25,25 @@ public function testHttpClientNoDebug()
         $container = static::$kernel->getContainer();
         $this->assertTrue($container->has('httplug.client'));
         $client = $container->get('httplug.client');
-        $this->assertInstanceOf('Http\Client\HttpClient', $client);
+        $this->assertInstanceOf(HttpClient::class, $client);
+    }
+
+    public function testDebugToolbar()
+    {
+        static::bootKernel(['debug' => true]);
+        $container = static::$kernel->getContainer();
+        $this->assertTrue($container->has('profiler'));
+        $profiler = $container->get('profiler');
+        $this->assertInstanceOf(Profiler::class, $profiler);
+        $this->assertTrue($profiler->has('httplug'));
+        $collector = $profiler->get('httplug');
+        $this->assertInstanceOf(DebugPluginCollector::class, $collector);
+        /** @var PluginJournal $journal */
+        $journal = $collector->getJournal();
+        $plugins = $journal->getPlugins('acme');
+        $this->assertEquals([
+            'httplug.plugin.stopwatch',
```

Comment:
@Nyholm its expected that the journal also tracks the stopwatch, right? or should we hide that from the journal?